packages/components/src/Menu/Menu.css

```diff
@@ -10,28 +10,18 @@
 .menu {
   position: fixed;
   right: 0;
+  bottom: 0;
+  left: 0;
   z-index: var(--elevation-menu);
   width: 100%;
+  max-height: 72vh;
   padding: var(--space-small);
+  padding-bottom: env(safe-area-inset-bottom);
   border: var(--border-base) solid var(--color-grey--lighter);
+  border-radius: var(--radius-base) var(--radius-base) 0 0;
+  overflow-y: scroll;
   background-color: var(--color-white);
-}
-
-@media only screen and (max-width: 639px), print {
-  .menu {
-    bottom: 0;
-    left: 0;
-    max-height: 72vh;
-    padding-bottom: env(safe-area-inset-bottom);
-    border-radius: var(--radius-base) var(--radius-base) 0 0;
-    overflow-y: scroll;
-    transition: transform 300ms ease;
-    -webkit-overflow-scrolling: touch;
-  }
-
-  .menu:last-child {
-    padding-bottom: var(--space-small);
-  }
+  -webkit-overflow-scrolling: touch;
 }
 
 @media only screen and (min-width: 640px), print {
```

Comment:
I think we need to set an explicit right value now on these medium+ screens to get the menus aligning to the right of the button as they were prior

Reply:
(also pushed up)

Reply:
the `left: auto` should handle that I believe

Reply:
Yep, added that in below 👍